typescript/introduction/tsc-variable-declaration/tsc-var.md

```diff
@@ -0,0 +1,60 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Declaring Variables
```

Comment:
similarily, I think this is more of a generic JS insight than a TS one - maybe we can mention the `var/let` difference in one insight instead presenting all their technical differences?